src/js/components/stock-movement/modals/RejectRequestModal.scss

```diff
@@ -0,0 +1,119 @@
+.reject-request-modal-content {
+  position: absolute;
+  top: 20%;
+  left: 23%;
+  background-color: #ffffff;
+  width: 55vw;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.reject-request-modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.45);
+}
+
+// Classic form
+.classic-form {
```

Comment:
as much as I would love to use some bootstrap styling or global classes here I understand that it is hard so this can stay.
I was hoping that we can use `.panel-body .classic-form` classes which are defined on stockMovement.scss but it is written specifically for that page, unfortunately...
If you can do anything to shorten this CSS and maybe use bootstrap classes in some places then it would be great, if not then I won't force you to, unless you want to define some global styles for form input yourself which we will be able to sue in the future.